src/shared-api-objects/user-has-access-react.component.tsx

```diff
@@ -6,7 +6,7 @@ export default function UserHasAccessReact(props) {
 
   React.useEffect(() => {
     const subscription = getCurrentUser({ includeAuthStatus: false }).subscribe(
-      x => {
+      (x: any) => {
```

Comment:
What breaking change are you talking about with 1.0.4? The build passed - https://travis-ci.org/openmrs/openmrs-esm-api/builds/592146027, and it looks like openmrs-spa.org is still working, too.

Reply:
This addresses the typescript error